### PR TITLE
Migration to package format 2

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: schunk_powercube_chain/CMakeLists.txt [hunk 1]
```diff
@@ -25,12 +27,4 @@ install(TARGETS ${PROJECT_NAME}
 
 install(DIRECTORY common/include/${PROJECT_NAME}/
    DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-   FILES_MATCHING PATTERN "*.h"
-   PATTERN ".svn" EXCLUDE
-)
-
-install(DIRECTORY orocos/include/${PROJECT_NAME}/
```
Comment (NadiaHG) [inline schunk_powercube_chain/CMakeLists.txt:32]: is orocos not needed anymore ?

**Comment by mgruhler** [comment]
:question: can't say anything about this one...

File: schunk_powercube_chain/CMakeLists.txt (hunk 1 above)
Comment (fmessmer) [inline schunk_powercube_chain/CMakeLists.txt:32]: never has...

**Comment by NadiaHG** [comment]
LGTM

**Pull request merged**